scripts/zones/Jugner_Forest/mobs/Giollemitte_B_Feroun.lua

```diff
@@ -0,0 +1,59 @@
+----------------------------------
+-- Area: Jugner_Forest
+--   NM: Giollemitte B Feroun
+-- Involved in Quest: A Timely Visit
+-----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+
+function onMobInitialize(mob)
+
+    mob:setMobMod(dsp.mobMod.IDLE_DESPAWN, 300) -- 5 minutes
+    mob:setMobMod(dsp.mobMod.ADD_EFFECT, 1)
+    
+end
+
+function onMobSpawn(mob)
+
```

Comment:
Couple of things to work out here:

- you need to rebase, there are conflicts preventing merge. Sorry this happens sometimes

- the extra newlines make for weird formatting.

- the old version of the 2hr scripting is still there commented out. remove it entirely please.

- you have some `addMod()` in the `onMobSpawn()` that need explained or removed. Also they should be in `onMobInitialize()` instead, if and only if they are absolutely necessary. The mobs Job will normally give it traits that will add some mods. From what I see on wiki, this mob is a paladin. paladins already come with resist sleep baked in (and it applies to multiple forms of sleep). They don't have magic defense bonus, but if you know this guy does on retail for a fact that indicates to me either a rdm or whm sub granting that trait is likely. Wiki says "seems to".  Wiki being correct with no data is like a coin toss. I'm willing to go with it till someone proves it right or wrong, but it's probably from a job trait rather than non standard stacks tacked on. (note: those DO happen, they just aren't very common)